### PR TITLE
feat : [ARLS] Enabling sbl resiliancy

### PR DESCRIPTION
**Pull request by aborwank**
Changing TcoBase for Tco timer for sbl resiliancy

### CODE REVIEW

File: Platform/ArrowlakeBoardPkg/Script/StitchIfwiConfig_arls.py [hunk 1]
```diff
@@ -147,8 +147,9 @@ def get_xml_change_list (platform, plt_params_list):
         # Path                                                                      | value |
         # =========================================================================================
         #   Region Order
-        ('./FlashSettings/BiosConfiguration/PchTopSwapOverride',                                    '512KB'),
-        ('./FlashSettings/BiosConfiguration/SocTopSwapOverride',                                    '512KB'),
+        ('./FlashSettings/BiosConfiguration/PchTopSwapOverride',                                    '4MB'),
```
Comment (lsubash) [inline Platform/ArrowlakeBoardPkg/Script/StitchIfwiConfig_arls.py:150]: For Topswap settings, try using patch_xml_file() available in StitchIFwi for ARL-S.

File: Silicon/ArrowlakePkg/Include/Register/TcoRegs.h [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+#define R_TCO_IO_TCO2_CNT                            0x0A
+#define S_TCO_IO_TCO2_CNT                            2
+#define N_TCO_IO_TCO2_CNT_INTRD_SEL                  2
+
+#define R_TCO_IO_TWDS                                0x0E
+#define R_TCO_IO_TMR                                 0x12
+#define B_TCO_IO_TMR                                 0xBFF
+
+#define TCO_BASE_ADDRESS                             0x8E00
```
Comment (lsubash) [inline Silicon/ArrowlakePkg/Include/Register/TcoRegs.h:82]: Later, check to get the TCO BASE ADDREESS from common code.
We can use the same TCO time lib across platforms

**Review by lsubash — APPROVED** [review]
Minor comments.

**Review by lsubash — APPROVED** [review]
Looks good... fix comments later

**Review by gdong1 — APPROVED** [review]

**Pull request merged**